templates/stanford_event/stanford-event.html.twig

```diff
@@ -59,7 +24,13 @@
         {% if where|render|striptags|trim is not empty %}
         <dt>{{ 'Where'|t }}:</dt>
         <dd>
-          {{ where }}
+          {% if mapurl|render|striptags|trim is not empty %}
+          <A href="{{ mapurl }}">
```

Comment:
May was well output `{{ mapurl|render|striptags|trim }}` inside the href property.